apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/DmlTests.cs

```diff
@@ -531,9 +530,140 @@ public void PartitionedUpdate_Medium()
                 {
                     command.Parameters.Add("key", SpannerDbType.String, key);
                     command.Parameters.Add("cutoff", SpannerDbType.Int64, amountToAdd);
-                    Assert.Equal(0, (long)command.ExecuteScalar());
+                    Assert.Equal(0, (long) command.ExecuteScalar());
                 }
             }
         }
+
+        [Fact(Skip = "This test is skipped until DML Returning changes are done.")]
+        public async Task DMLReturn_ExecuteReader_Read()
+        {
+            string key = _fixture.CreateTestRows();
+            var actualValues = new List<int>();
+
+            using var connection = _fixture.GetConnection();
+            {
+                await connection.OpenAsync();
+                RetryHelpers.ExecuteWithRetry(async () =>
+                {
+                    string dml = $"UPDATE {_fixture.TableName} SET Value = OriginalValue + 1 WHERE UpdateMe AND Key=@key Then Return Value";
+                    var command = connection.CreateDmlCommand(dml);
+                    using (var reader = await command.ExecuteReaderAsync())
```

Comment:
You could use the simpler `using` statement format here as well to reduce indentation a tiny bit.